### PR TITLE
Update EnableBranchingModal.tsx

### PR DESCRIPTION
**Pull request by MildTomato**
## What kind of change does this PR introduce?

- updates pricing copy to be more clearer in opt in UI for branching

## What is the current behavior?

Please link any relevant issues here.

## What is the new behavior?

Feel free to include screenshots if it includes visual changes.

## Additional context

Add any other context or screenshots.

### CODE REVIEW

File: apps/studio/components/layouts/AppLayout/EnableBranchingButton/EnableBranchingModal.tsx [hunk 1]
```diff
@@ -234,12 +234,11 @@ const EnableBranchingModal = () => {
...
                     <div className="flex flex-col gap-y-1">
                       <p className="text-sm text-foreground">
-                        Preview branches are billed per hour
+                        Preview branches are billed $0.32 per day (approximately $10 per month)
                       </p>
                       <p className="text-sm text-foreground-light">
-                        Launching a new preview branch incurs additional compute costs, starting at
-                        $0.01344 per hour (~$10/month). This cost will continue for as long as the
-                        branch has not been removed.
+                        Launching a new preview branch incurs additional compute costs at $0.32 per
```
Comment (kevcodez) [inline apps/studio/components/layouts/AppLayout/EnableBranchingButton/EnableBranchingModal.tsx:240]: The line break looks pretty odd here, did you run prettier?

**Review by kevcodez — APPROVED** [review]

File: apps/docs/pages/guides/platform/branching.mdx [hunk 2]
```diff
@@ -627,6 +627,15 @@ If you prefer not to use branching, you can manage your environments and tests i
...
 You can also combine both strategies to perform both local and staging tests.
 
+## Early Access Pricing
+
+Branching is available for Early Access, available on the Pro Plan and above.
+
+The Early Access pricing is as follows:
+
+- Each Preview branch is $0.32 per day
+- Each Preview branch is billed until it is removed
```
Comment (charislam) [inline apps/docs/pages/guides/platform/branching.mdx:637]:
```suggestion
Branching is available on the Pro Plan and above for Early Access users. The price during Early Access is:

- Each Preview branch costs $0.32 per day
- Each Preview branch is billed until it is removed
```

File: apps/studio/components/interfaces/BranchManagement/CreateBranchModal.tsx [hunk 3]
```diff
@@ -217,11 +217,10 @@ const CreateBranchModal = ({ visible, onClose }: CreateBranchModalProps) => {
           <Modal.Content className="py-2">
             <Alert_Shadcn_ variant="warning">
               <IconAlertCircle strokeWidth={1.5} />
-              <AlertTitle_Shadcn_>Preview branches are billed per hour</AlertTitle_Shadcn_>
+              <AlertTitle_Shadcn_>Preview branches are billed $0.32 per day</AlertTitle_Shadcn_>
```
Comment (charislam) [inline apps/studio/components/interfaces/BranchManagement/CreateBranchModal.tsx:220]:
```suggestion
              <AlertTitle_Shadcn_>Each preview branches costs $0.32 per day</AlertTitle_Shadcn_>
```

File: apps/studio/components/interfaces/BranchManagement/CreateBranchModal.tsx [hunk 4]
```diff
@@ -217,11 +217,10 @@ const CreateBranchModal = ({ visible, onClose }: CreateBranchModalProps) => {
...
               <AlertDescription_Shadcn_>
-                Launching a new preview branch incurs additional compute costs, starting at $0.01344
-                per hour (~$10/month). This cost will continue for as long as the branch has not
-                been removed.
+                Launching a new preview branch incurs additional compute costs of $0.32 per day.
+                This cost will continue for as long as the branch has not been removed.
```
Comment (charislam) [inline apps/studio/components/interfaces/BranchManagement/CreateBranchModal.tsx:223]:
```suggestion
                Each preview branch costs $0.32 per day until it is removed.
```

**Review by charislam — COMMENTED** [review]
suggestions only to shorten up the copy and make it more "plain English"-y

**Pull request merged**